autogpt_platform/backend/backend/blocks/io.py

```diff
@@ -0,0 +1,515 @@
+        )
+
+    class Output(AgentInputBlock.Output):
+        result: str = SchemaField(description="Short text result.")
+
+    def __init__(self):
+        super().__init__(
+            id="7fcd3bcb-8e1b-4e69-903d-32d3d4a92158",
+            description="Block for short text input (single-line).",
+            input_schema=ShortTextAgentInputBlock.Input,
+            output_schema=ShortTextAgentInputBlock.Output,
+            test_input=[
+                {
+                    "value": "Hello",
+                    "name": "short_text_1",
+                    "description": "Short text example 1",
+                    "placeholder_values": [],
+                    "limit_to_placeholder_values": False,
+                },
+                {
+                    "value": "Quick test",
+                    "name": "short_text_2",
+                    "description": "Short text example 2",
+                    "placeholder_values": ["Quick test", "Another option"],
+                    "limit_to_placeholder_values": True,
+                },
+            ],
+            test_output=[
+                ("result", "Hello"),
+                ("result", "Quick test"),
+            ],
+        )
+
+
+class LongTextAgentInputBlock(AgentInputBlock):
+    class Input(AgentInputBlock.Input):
+        value: str = SchemaField(
+            description="Long text input (potentially multi-line).",
+            default="Longer text can go here.",
+            json_extra={"format": "long-text"},
+        )
+
+    class Output(AgentInputBlock.Output):
+        result: str = SchemaField(description="Long text result.")
+
+    def __init__(self):
+        super().__init__(
+            id="90a56ffb-7024-4b2b-ab50-e26c5e5ab8ba",
+            description="Block for long text input (multi-line).",
+            input_schema=LongTextAgentInputBlock.Input,
+            output_schema=LongTextAgentInputBlock.Output,
+            test_input=[
+                {
+                    "value": "Lorem ipsum dolor sit amet...",
+                    "name": "long_text_1",
+                    "description": "Long text example 1",
+                    "placeholder_values": [],
+                    "limit_to_placeholder_values": False,
+                },
+                {
+                    "value": "Another multiline text input.",
+                    "name": "long_text_2",
+                    "description": "Long text example 2",
+                    "placeholder_values": ["Another multiline text input."],
+                    "limit_to_placeholder_values": True,
+                },
+            ],
+            test_output=[
+                ("result", "Lorem ipsum dolor sit amet..."),
+                ("result", "Another multiline text input."),
+            ],
+        )
+
+
+class NumberAgentInputBlock(AgentInputBlock):
+    class Input(AgentInputBlock.Input):
+        value: int = SchemaField(
+            description="Number input.",
+            default=42,
```

Comment:
I don't think this should have a default value

Reply:
Removed